src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import './Tabs.css';
+import PropTypes from 'prop-types';
+import Tab from '../Tab/Tab';
+
+class Tabs extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      index: 0,
+    };
+    this.handleIndex = this.handleIndex.bind(this);
+  }
+
+  handleIndex(title) {
```

Comment:
handleIndex? does this function do something with index, is this its main purpose?

Reply:
use index of an element as argument, not title